include/onnxruntime/core/optimizer/graph_transformer_utils.h

```diff
@@ -80,6 +81,7 @@ InlinedVector<std::unique_ptr<GraphTransformer>> GenerateTransformersForMinimalB
     const IExecutionProvider& cpu_execution_provider,
     const InlinedHashSet<std::string>& rules_and_transformers_to_disable = {});
 
+bool IsFp16Supported(KernelRegistry* kernelRegistry);
```

Comment:
add documentation for function.

also, maybe consider a more specific name like `IsFp16FusedConvSupported`